src/usr/local/www/system_advanced_network.php

```diff
@@ -186,6 +199,15 @@
 ))->setHelp('By default, if IPv6 is configured and a hostname resolves IPv6 and IPv4 addresses, '. 
 	'IPv6 will be used. If this option is selected, IPv4 will be preferred over IPv6.');
 
+$section->addInput(new Form_Input(
+	'global-v6duid',
+	'DHCP6 DUID',
+	'text',
+	$pconfig['global-v6duid'],
+	['placeholder' => 'xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx']
+	))->setWidth(9)->sethelp('Enter the DUID to use here. Use this option if using RAM Disk, as the DUID will be lost on reboot. The existing DUID may be found in var/db/dhcp6_duid.' .'<br />' .
```

Comment:
If they leave the field blank it will use the existing DUID, which is what the placeholder is at the time. There wouldn't be a way to make it editable from there but usually it wouldn't need to be. If someone is changing it they likely would be changing it to a much different value.